tests/api-test/createSecret.bru

```diff
@@ -19,3 +19,34 @@ body:json {
 body:text {
   "Super secret value"
```

Comment:
we might want to have the automation script or phony target in the Makefile such as:
make api-test
and we start automation, can we wor on that?

Reply:
I have checked how can we automate running Bruno tests. We would need access to NPM.
More info here: https://docs.usebruno.com/bru-cli/overview